src/main/java/pl/wedding/management/service/reservation/domain/converter/RowFactory.java

```diff
@@ -0,0 +1,15 @@
+package pl.wedding.management.service.reservation.domain.converter;
+
+import java.util.function.Function;
+
+abstract class RowFactory<DTO, ENTITY> {
```

Comment:
Jeżeli w tej klasie nie ma żadnej innej metody jak ta abstrakcyjna convertFromDTO() to klasa jest niepotrzebna. Możesz z tego zrobić interfejs.